Week1/homework/todo sql.sql

```diff
@@ -0,0 +1,42 @@
+CREATE DATABASE IF NOT EXISTS  base_todoinfo;
+
+USE base_todoinfo;
+
+CREATE TABLE IF NOT EXISTS todos(
+	id_todo INT NOT NULL,
```

Comment:
(this was not part of the homework... but while we are at it) 
convention is we only call it id, any time it will get referenced it will be prefixed by `todos`. `id_todo` looks like a forign key.. which we normally use `todo_id`.  If we retrieve a single row its a todo, so if we want the id its just `todo.id` or if we use the id to join table we write the whole `on todos.id = other_entity.todo_id`

Reply:
oh and while we are at it:
`id int NOT NULL AUTO_INCREMENT ` -> to get the auto increment id (well see how this helps when inserting data next class)